frontend/src/theme.ts

```diff
@@ -0,0 +1,32 @@
+import { createMuiTheme } from '@material-ui/core';
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: '#6d326d',
+      light: '#9d5e9b',
+      dark: '#400442',
+    },
+    secondary: {
+      main: '#5da9e9',
+      light: '#93daff',
+      dark: '#187ab6',
+    },

```

Comment:
Idk about you, but I like the current dark purple theme alot!

Reply:
👍